cityofaustin/atd-moped
# Drop work assignment ID column

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/10954

## Testing
**URL to test:** Local


**Steps to test:**

We're dropping a column which was already hidden from view of users, so testing is focused on making sure the app still works normally.

1. Start the hasura cluster and editor
2. Verify that the project lists loads
3. Open a few projects to ensure they load correctly
4. Create a new project



---
#### Ship list
